includes/class-wc-gateway-swedbank-pay-checkout.php

```diff
@@ -0,0 +1,1837 @@
+		}
+
+		// Checkout scripts
+		if ( is_checkout() || isset( $_GET['pay_for_order'] ) || is_add_payment_method_page() ) {
+			if ( $this->instant_checkout === 'yes' ) {
+				wp_register_script( 'wc-gateway-swedbank-pay-checkout', untrailingslashit( plugins_url( '/', __FILE__ ) ) . '/../assets/js/checkout' . $suffix . '.js', [
+					'jquery',
+					'wc-checkout'
+				], false, true );
+			} else {
+				// Styles
+				wp_enqueue_script( 'featherlight', untrailingslashit( plugins_url( '/', __FILE__ ) ) . '/../assets/js/featherlight/featherlight' . $suffix . '.js', [ 'jquery' ], '1.7.13', true );
+				wp_enqueue_style( 'featherlight-css', untrailingslashit( plugins_url( '/', __FILE__ ) ) . '/../assets/js/featherlight/featherlight' . $suffix . '.css', [], '1.7.13', 'all' );
+
+				wp_register_script( 'wc-gateway-swedbank-pay-checkout', untrailingslashit( plugins_url( '/', __FILE__ ) ) . '/../assets/js/checkout' . $suffix . '.js', [
+					'jquery',
+					'wc-checkout',
+					'featherlight'
+				], false, true );
+			}
+
+			// Localize the script with new data
+			$translation_array = [
+				'culture'                      => $this->culture,
+				'instant_checkout'             => ( $this->instant_checkout === 'yes' ),
+				'needs_shipping_address'       => WC()->cart->needs_shipping(),
+				'ship_to_billing_address_only' => wc_ship_to_billing_address_only(),
+				'checkin'                      => ( $this->checkin === 'yes' ),
+				'nonce'                        => wp_create_nonce( 'swedbank_pay_checkout' ),
+				'ajax_url'                     => admin_url( 'admin-ajax.php' ),
+				'paymentMenuStyle'             => null,
+				'checkInStyle'                 => null,
+			];
+
+			// Add PM styles
+			if ( $styles = apply_filters( 'swedbank_pay_checkout_paymentmenu_style', $this->paymentMenuStyle ) ) {
+				$translation_array['paymentMenuStyle'] = $styles;
+			}
+
+			// Add CheckIn Styles
+			if ( $styles = apply_filters( 'swedbank_pay_checkout_checkin_style', $this->checkInStyle ) ) {
+				$translation_array['checkInStyle'] = $styles;
+			}
+
+			wp_localize_script( 'wc-gateway-swedbank-pay-checkout', 'WC_Gateway_Swedbank_Pay_Checkout', $translation_array );
+
+			// Enqueued script with localized data.
+			wp_enqueue_script( 'wc-gateway-swedbank-pay-checkout' );
+		}
+	}
+
+
+	/**
+	 * If There are no payment fields show the description if set.
+	 */
+	public function payment_fields() {
+
+	}
+
+	/**
+	 * Validate Frontend Fields
+	 * @return bool|void
+	 */
+	public function validate_fields() {
+		//
+	}
+
+	/**
+	 * Thank you page
+	 *
+	 * @param $order_id
+	 */
+	public function thankyou_page( $order_id ) {
+		WC()->session->__unset( 'payex_paymentorder_id' );
+	}
+
+	/**
+	 * Process Payment
+	 *
+	 * @param int $order_id
+	 *
+	 * @return array|false
+	 */
+	public function process_payment( $order_id ) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method process_payment() has an NPath complexity of 65280. The configured NPath complexity threshold is 200.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)